nuxeo-retention-web/elements/nuxeo-retention-analytics.js

```diff
@@ -0,0 +1,351 @@
+            data="{{noRetention}}"
+            index="[[index]]"
+          >
+          </nuxeo-repository-data>
+
+          <!-- Future Retentions -->
+          <nuxeo-repository-data
+            start-date="[[_formatDate(startDate)]]"
+            end-date="[[_extendEndDate(endDate)]]"
+            date-field="dc:created"
+            grouped-by="ecm:retainUntil"
+            group-limit="10"
+            data="{{futureRetention}}"
+            index="[[index]]"
+          >
+          </nuxeo-repository-data>
+
+          <!-- Expired Retentions -->
+          <nuxeo-repository-data
+            start-date="[[_formatDate(startDate)]]"
+            end-date="[[_extendEndDate(endDate)]]"
+            date-field="dc:created"
+            grouped-by="record:retainUntil"
+            group-limit="10"
+            data="{{expiredRetention}}"
+            index="[[index]]"
+          >
+          </nuxeo-repository-data>
+
+          <nuxeo-card heading="[[i18n('retentionAnalytics.retentionStatus')]]">
+            <chart-pie
+              values="[[_valuesRetentionStatus(noRetention, futureRetention, expiredRetention)]]"
+              labels="[[_labelsRetentionStatus()]]"
+              options='{ "legend": { "display": true, "position": "bottom", "labels": { "boxWidth": 12 } }, "animation": false }'
+            >
+            </chart-pie>
+          </nuxeo-card>
+
+          <!-- LegalHold by Document Type -->
+          <nuxeo-repository-data
+            start-date="[[_formatDate(startDate)]]"
+            end-date="[[_extendEndDate(endDate)]]"
+            date-field="dc:created"
+            grouped-by="ecm:primaryType"
+            where='{"ecm:hasLegalHold": "true"}'
+            group-limit="20"
+            data="{{legalHoldByDocType}}"
+            index="[[index]]"
+          >
+          </nuxeo-repository-data>
+
+          <nuxeo-card heading="[[i18n('retentionAnalytics.legalHoldByDocType')]]">
+            <chart-pie
+              values="[[_values(legalHoldByDocType)]]"
+              labels="[[_labels(legalHoldByDocType)]]"
+              options='{ "legend": { "display": true, "position": "bottom", "labels": { "boxWidth": 12 } }, "animation": false }'
+            >
+            </chart-pie>
+          </nuxeo-card>
+        </div>
+
+        <!-- Expiration next 12 months -->
+        <nuxeo-repository-data
+          start-date="[[_formatDateNow()]]"
+          end-date="[[_formatDateFutureInMonths(12)]]"
+          with-date-intervals="month"
+          date-field="ecm:retainUntil"
+          data="{{expirationPerMonth}}"
+          index="[[index]]"
+        >
+        </nuxeo-repository-data>
+
+        <nuxeo-card class="chartBar" heading="[[i18n('retentionAnalytics.expirationNext12Months.heading')]]">
+          <chart-bar
+            labels="[[_futureExpirationLabels(expirationPerMonth)]]"
+            values="[[_values(expirationPerMonth)]]"
+            style="min-width:800px;"
```

Comment:
we should avoid in-line styling... can we try another solution here?

Reply:
yes, removed.